Dockerfile

```diff
@@ -78,6 +83,7 @@ RUN zypper refresh \
         wget \
         which \
         xz-devel \
+        yq \
         zlib-devel \
         && zypper clean -a \
         && suseconnect --cleanup \
```

Comment:
We can leave off this trailing slash.

Reply:
Yeah that should go.
I'm surprised it's even here.